rskj-core/src/test/java/co/rsk/peg/BridgeSerializationUtilsTest.java

```diff
@@ -94,6 +96,42 @@ class BridgeSerializationUtilsTest {
     private static final Address ADDRESS = BitcoinTestUtils.createP2PKHAddress(MAINNET_PARAMETERS, "first");
     private static final Address OTHER_ADDRESS = BitcoinTestUtils.createP2PKHAddress(MAINNET_PARAMETERS, "second");
 
+
+    private static Stream<Arguments> validRskAddressesProvider() {
+        return Stream.of(
+            Arguments.of(TestUtils.generateAddress("address1")),
+            Arguments.of(TestUtils.generateAddress("address2")),
+            Arguments.of(TestUtils.generateAddress("address3"))
+        );
+    }
+
+    @ParameterizedTest
+    @MethodSource("validRskAddressesProvider")
+    void serializeAndDeserializeRskAddress_validAddress_ok(RskAddress address) {
+        // Act
+        byte[] actualSerializedAddress = BridgeSerializationUtils.serializeRskAddress(address);
+        RskAddress actualDeserializedAddress = BridgeSerializationUtils.deserializeRskAddress(actualSerializedAddress);
+
+        // Assert
+        assertNotNull(actualSerializedAddress);
```

Comment:
```suggestion
        byte[] actualSerializedAddress = BridgeSerializationUtils.serializeRskAddress(address);
        assertNotNull(actualSerializedAddress);
        RskAddress actualDeserializedAddress = BridgeSerializationUtils.deserializeRskAddress(actualSerializedAddress);

        // Assert
```

no big deal, but i think is looks better to assert the serialized is not null before deserializing it